RxCocoa/iOS/UIScrollView+Rx.swift

```diff
@@ -77,6 +77,22 @@
             return ControlEvent(events: source)
         }
         
+        /// Reactive wrapper for delegate method `scrollViewWillEndDragging(_:withVelocity:targetContentOffset:)`
+        public var willEndDragging: ControlEvent<(CGPoint, UnsafeMutablePointer<CGPoint>?)> {
```

Comment:
Hi,

It seems to me that this is the signature
```
optional public func scrollViewWillEndDragging(_ scrollView: UIScrollView, withVelocity velocity: CGPoint, targetContentOffset: UnsafeMutablePointer<CGPoint>)
```

so that would mean we would want to have
```
public var willEndDragging: ControlEvent<(CGPoint, UnsafeMutablePointer<CGPoint>)> 
```

I guess you can throw `RxCocoaError.unknown` otherwise and turn `if let rawPointer = targetContentOffsetValue.pointerValue` into a `guard`.  ¯\\\_(ツ)\_/¯